packages/components/tsup.config.ts

```diff
@@ -1,14 +1,19 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import { defineConfig } from "tsup"
+import cssModules from "./plugins/esbuild-css-module"
 
 export default defineConfig({
   entry: ["src/**/*.ts"],
-  format: ["cjs", "esm", "iife"],
+  outDir: "./dist",
+  format: ["cjs", "esm"],
```

Comment:
Oh yay you got rid of the weird iife that no one understood :P

Reply:
Yeahhh it actually took the longest to compile and it didn't actually work with the SCSS files referencing `../variables` files haha